Remove trailing spaces and reduce log verbosity in parallel.py

sickpig opened the pull request.

sickpig committed:
Remove trailing spaces from parallel.py

awemany reviewed (approved):
```
$ git show -w pr1249
commit 7cb1167ed48139987814032d61862404ec86449a (pr1249)
Author: Andrea Suisani <sickpig@gmail.com>
Date:   Wed Aug 8 12:10:13 2018 +0200

    Remove trailing spaces from parallel.py

```

sickpig committed:
Reduce log verbosity for parallel.py

Previously all debug categories were enabled due to the fact we were
hunting an intermittent failure. The test is very stable now so there's
no need for such level of details in the log.

sickpig commented:
added a new commit to reduce the log verbosity

awemany reviewed (commented):
Typo in commit headline: 'prallel.py'.

Other than that, looks good. The lines you changed are awful copy-n-paste galore, though, so IMO it would be great if *someone* could clean them up :D
